luaui/Widgets/gui_anti_ranges.lua

```diff
@@ -70,20 +62,34 @@ local GetUnitIsStunned     		= Spring.GetUnitIsStunned
 
 local antiInLos					= {}
 local antiOutLos				= {}
-
-local coverageRangeArmStatic	= WeaponDefs[UnitDefNames.armamd.weapons[1].weaponDef].coverageRange
-local coverageRangeCoreStatic	= WeaponDefs[UnitDefNames.corfmd.weapons[1].weaponDef].coverageRange
-local coverageRangeArm			= WeaponDefs[UnitDefNames.armscab.weapons[1].weaponDef].coverageRange
-local coverageRangeCore			= WeaponDefs[UnitDefNames.cormabm.weapons[1].weaponDef].coverageRange
-local coverageRangeArmWater		= WeaponDefs[UnitDefNames.armcarry.weapons[1].weaponDef].coverageRange
-local coverageRangeCoreWater	= WeaponDefs[UnitDefNames.corcarry.weapons[1].weaponDef].coverageRange
-local coverageRangeArmWater2	= WeaponDefs[UnitDefNames.armantiship.weapons[1].weaponDef].coverageRange
-local coverageRangeCoreWater2	= WeaponDefs[UnitDefNames.corantiship.weapons[1].weaponDef].coverageRange
+local antiNukeDefs              = {}
 
 local diag = math.diag
 
 local chobbyInterface
 
+--------------------------------------------------------------------------------
+-- Initialization
+--------------------------------------------------------------------------------
+
+function widget:Initialize()
+    identifyAntiNukeUnits()  -- Pre-process unit definitions
```

Comment:
```suggestion
    identifyAntiNukeUnits()
```
Comments not needed when code is self-documenting